config/core/runtime-configs.yaml

```diff
@@ -11,17 +11,8 @@ runtimes:
     queue_timeout:
       days: 1
     filters:
+      - regex: {'tree': '^(staging|mainline|next|stable|arm64|riscv|soc|amlogic|matthiasbgg)$'}
```

Comment:
There's actually no `staging` tree, staging jobs use the `kernelci` tree (I know it's been in the YAML for ages, but spotting this now).

Reply:
> I pushed an updated version with `s/staging/kernelci/`

I don't see it in the diff, was it missing from your commit maybe?